dev-docs/analytics/adWMG.md

```diff
@@ -3,9 +3,29 @@ layout: analytics
 title: adWMG
 description: adWMG Analytics Adapter
 modulecode: adWMG
+gdpr_supported: true
+usp_supported: false
+coppa_supported: false
+prebid_member: true
+gvl_id: 959
 ---
 
-#### Registration
+#### Analytics Options
 
-Please visit [https://adwmg.com/](https://adwmg.com/) for more information.
```

Comment:
So you don't want any contact info at all in this?

Reply:
@bretg Oh, thanks, this info was lost accidentally. Added it now